styles.css

```diff
@@ -114,3 +114,417 @@ p {
+  font-size: 17px;
+  line-height: 24px;
+
+  /* display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em; */
+}
+
+/* backgroud picture and stack */
+
+/* background effect */
+
+.data-arc {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  height: 87px;
+  padding-left: 24px;
+  width: 100%;
+}
+
+.see-project-btn {
+  position: relative;
+  width: 100%;
+  height: 48px;
+  left: 0;
+  top: 0;
+  background: #ff6b00;
+  color: #fff;
+  border: #fff;
+}
+
+.details {
+  background-image: url(images/bg-img.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  color: white;
+  padding-top: 20%;
+  margin: 35px;
+}
+
+/* about me section */
+
+#about-me-sec {
+  background-color: #1c1a19;
+  color: #fff;
+  align-items: center;
+  text-align: center;
+}
+
+.about-me {
+  background-image: url(images/about-me.png);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  padding-right: 24px;
+  padding-bottom: 220px;
+  align-items: flex-start;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: left;
+}
+
+.get-my-resume-btn {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  background: #ff6b00;
+  border: #fff;
+  text-align: left;
+  padding: 12px;
+  padding-left: 20px;
+  margin-left: 25px;
+}
+
+#color {
+  margin-left: 15px;
+  color: #091e42;
+}
+
+.vertical-list,
+.stack-Skills {
+  align-items: center;
+  text-align: center;
+}
+
+/* .about-me {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: left;
+} */
+
+.about-header-txt {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  color: #ff6b00;
+  padding: 30px 0 0 20px;
+}
+
+.about-txt {
+  padding: 30px 0 0 20px;
+  padding-bottom: 20px;
+  margin-bottom: 20px;
+}
+
+.vert-title {
+  font-family: 'InaiMathi', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+/* .lang-stack {
+    font-family: 'Inter';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    text-align: center;
+    align-items: center;
+    padding-left: 120px;
+
+}
+.frame {
+    font-family: 'Inter';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    text-align: center;
+    align-items: center;
+    padding-left: 25px;
+}
+.skill-list {
+    font-family: 'Inter';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    text-align: center;
+    align-items: center;
+    padding-left: 25px; */
```

Comment:
- [ ] kindly consider removing the unwanted piece of codes  they make your codebase look messy and this will help it look more clean

Reply:
removed